core.py

```diff
@@ -80,16 +106,18 @@ def __init__(self, tof_lib):
         else:
             self.arduino = None
             self.PWMservo = PWM.Servo(pulse_incr_us=1)
-            for pin in range(0, 3):
-                i2c_lidar.xshut([LIDAR_PINS[pin]])
-                self.lidars.append(
-                    i2c_lidar.create(LIDAR_PINS[pin], tof_lib, 0x2a + pin)
-                )
+
+        # I2C lidar sensors should be enabled
+        # in both Arduino mode or direct pi mode.
+        for pin in range(0, 3):
+            i2c_lidar.xshut([LIDAR_PINS[pin]])
```

Comment:
well, that's even better! Also, if the function is now misnamed, then *definitely* change it to something more meaningful or future travellers will be all kinds of confused. It'll only take a minte and future us will be grateful.